Makefile

```
@@ -157,8 +157,13 @@ kind-deploy-controller: generate-operator-yaml install-resources deploy
 .PHONY: deploy-controller
 deploy-controller: kind-deploy-controller
 
+HOSTED ?= none
+
 .PHONY: kind-deploy-controller-dev
-kind-deploy-controller-dev: kind-deploy-controller build-images
+kind-deploy-controller-dev:
+	if [ "$(HOSTED)" != "hosted" ]; then\
+		make kind-deploy-controller;\
```

For safety/consistency, this needs to use the `$(MAKE)` built-in variable:
https://www.gnu.org/software/make/manual/make.html#MAKE-Variable
```suggestion
		$(MAKE) kind-deploy-controller;\
```